fjord/suggest/providers/trigger/provider.py

```diff
@@ -61,15 +77,25 @@ def handle_redirect(self, request, redirect):
         except TriggerRule.DoesNotExist:
             return
 
+        # Interpolate the url.
+        destination_url = rule.url
+        if '{' in destination_url:
+            # Fetch the response so we can interpolate the url.
+            try:
+                response = FeedbackResponse.objects.get(id=response_id)
+            except FeedbackResponse.DoesNotExist:
+                return
```

Comment:
We could, but it seems pretty straight-forward and it's a very unlikely scenario. The only time feedback is deleted is if it's hate speech and I've noticed it.